src/exposure-provider.tsx

```diff
@@ -175,21 +170,10 @@ export const ExposureProvider: React.FC<ExposureProviderProps> = ({
         await configure();
         start();
       }
-
-      if (state.status.state === StatusState.unknown) {
-        clearAndResetTimer(unknownStatusTimer);
-        unknownStatusTimer.current = setTimeout(() => {
-          setState((s) => ({...s, initialised: true}));
-        }, 5000);
-      } else {
-        clearAndResetTimer(unknownStatusTimer);
-        setState((s) => ({...s, initialised: true}));
```

Comment:
If we're not going to set this to true we should probably remove it entirely
- from the`State` interface
- from the initial state
- from [the README](https://github.com/covidgreen/react-native-exposure-notification-service#initialised)

I guess this could be considered a breaking change as a few of the apps are relying on this value already?

Reply:
Yeah, if they can be removed, I would do that in a separate PR labeled semver-major

Reply:
Have added back support for setting the `initialised` state